library_generation/postprocess_library.sh

```diff
@@ -48,55 +51,33 @@ if [ $(pyenv virtualenvs | grep "${python_version}" | grep "postprocessing" | wc
 fi
 pyenv activate "postprocessing"
 
-# call owl-bot-copy
-owlbot_staging_folder="${postprocessing_target}/owl-bot-staging"
-mkdir -p "${owlbot_staging_folder}"
-echo 'Running owl-bot-copy'
-pre_processed_libs_folder=$(mktemp -d)
-# By default (thanks to generation templates), .OwlBot.yaml `deep-copy` section
-# references a wildcard pattern matching a folder
-# ending with `-java` at the leaf of proto_path. We then use a generated-java
-# folder that will be picked up by copy-code
-mkdir -p "${pre_processed_libs_folder}/${proto_path}/generated-java"
-copy_directory_if_exists "${preprocessed_sources_path}" "proto" \
-  "${pre_processed_libs_folder}/${proto_path}/generated-java/proto-google-cloud-library"
-copy_directory_if_exists "${preprocessed_sources_path}" "grpc" \
-  "${pre_processed_libs_folder}/${proto_path}/generated-java/grpc-google-cloud-library"
-copy_directory_if_exists "${preprocessed_sources_path}" "gapic" \
-  "${pre_processed_libs_folder}/${proto_path}/generated-java/gapic-google-cloud-library"
-copy_directory_if_exists "${preprocessed_sources_path}" "samples" \
-  "${pre_processed_libs_folder}/${proto_path}/generated-java/samples"
-pushd "${pre_processed_libs_folder}"
-# create an empty commit so owl-bot-copy can process this as a repo
-# (it cannot process non-git-repositories)
-git init
-git commit --allow-empty -m 'empty commit'
-popd # pre_processed_libs_folder
+if [[ -z "${owlbot_cli_source_folder}" ]]; then
+  owlbot_cli_source_folder=$(mktemp -d)
+  build_owlbot_cli_source_folder "${postprocessing_target}" "${owlbot_cli_source_folder}" "${preprocessed_sources_path}"
+fi
 
 owlbot_cli_image_sha=$(cat "${scripts_root}/configuration/owlbot-cli-sha" | grep "sha256")
 
+# we determine the location of the .OwlBot.yaml file by checking if the target
+# folder is a monorepo folder or not
+if [[ "${postprocessing_target}" == *google-cloud-java* ]]; then
+  owlbot_yaml_relative_path=".OwlBot.yaml"
+else
+  owlbot_yaml_relative_path=".github/.OwlBot.yaml"
```

Comment:
This looks like a tech debt to me, can we move `.OwlBot.yaml` to the root folder for all handwritten libraries? cc: @suztomo

Reply:
The filename is set to ".github/.OwlBot.yaml" as per the document go/owlbot-howto#githubowlbotyaml

Reply:
How did it work for google-cloud-java if it is not at the expected location under `.github`?

Reply:
@chingor13 Do you know what needed to have OwlBot service to read multiple OwlBot.yaml in non-standard location?